src/webgpu/util/device_pool.ts

```diff
@@ -238,6 +252,9 @@ class DeviceHolder implements DeviceProvider {
   state: DeviceHolderState = 'free';
   // initially undefined; becomes set when the device is lost
   lostInfo?: GPUDeviceLostInfo;
+  // Set if the device is expected to be lost. The null value is used to indicate that the lost
+  // reason can be anything.
```

Comment:
Why do we need to allow it to be anything? I would think null means that we''re not expecting device lost to happen. And if it's non-null, then we expect the reason to match exactly.

Reply:
Done.